include/exadg/poisson/spatial_discretization/operator.cpp

```diff
@@ -226,6 +226,24 @@ Operator<dim, Number, n_components>::setup(
   matrix_free      = matrix_free_in;
   matrix_free_data = matrix_free_data_in;
 
+  if(not(boundary_descriptor->dirichlet_cached_bc.empty()))
```

Comment:
Where is the Poisson coupled to other fields?

Reply:
Poisson is used as mesh motion algorithm / PDE model for the ALE-field in FSI. The displacement of the mesh at the FSI interface is prescribed by the structure (and is prescribed via a `DirichletCached` BC).